clients/include/testing_gemm.hpp

```diff
@@ -387,7 +388,10 @@ rocblas_status testing_gemm(Arguments argus)
         CHECK_HIP_ERROR(hipMemcpy(hC_2.data(), dC, sizeof(T) * size_C, hipMemcpyDeviceToHost));
 
         // CPU BLAS
-        cpu_time_used = get_time_us();
+        if(argus.timing)
```

Comment:
We do not need to collect timing info unless argus.timing is true. Previously the code unconditionally collected timing info, even if it didn't use it.